src/main/java/com/pracownia/vanet/view/Simulation.java

```diff
@@ -155,6 +165,11 @@ public void run() {
     private void drawDevices(Collection<Device> devices) {
         for (Device device : devices) {
             DeviceRepresentation representation = mapRepresentation.getRepresentation(device);
+            for (Integer id : this.fakeDevicesIds) {
```

Comment:
FakeDevices nie powinno być stanem klasy symulacji i jeśli miałby być tu używane to nie powinno być kopiowane tylko pobierane bezpośrednio z obiektów SIN.